### PR TITLE
Integration test protoc plugin

### PR DESCRIPTION
**Pull request by raboof**

### CODE REVIEW

File: src/test/java/wirelessmesh/IntegrationTest.java [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+import java.util.ArrayList;
+import java.util.Comparator;
+import java.util.List;
+import java.util.concurrent.ExecutionException;
+
+import static java.util.stream.Collectors.toList;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+@AkkaServerlessTest
```
Comment (pvlugter) [inline src/test/java/wirelessmesh/IntegrationTest.java]: This annotation is for JUnit 5 (to register the extension). Not needed for JUnit 4.

**Review by pvlugter — COMMENTED** [review]
Cool 👍🏼

File: src/test/java/wirelessmesh/IntegrationTest.java (hunk 1 above)
Comment (raboof) [reply src/test/java/wirelessmesh/IntegrationTest.java]: Ah, I didn't realize that. I wonder how we can make that discoverable. Perhaps we should just update the example project to JUnit 5.

Comment (raboof) [reply src/test/java/wirelessmesh/IntegrationTest.java]: (done in #2, not sure if it makes sense)

Comment (pvlugter) [reply src/test/java/wirelessmesh/IntegrationTest.java]: Yeah, not sure how to make it more clear. It's under a `junit.jupiter` package, and there are separate javadocs for `@AkkaServerlessTest`. Doesn't hurt to add the `@AkkaServerlessTest` annotation, just doesn't do anything.

**Comment by raboof** [comment]
Rebased as https://github.com/lightbend-labs/akkaserverless-wirelessmesh-java/pull/8

**Pull request closed without merging**